### ISSUE TITLE
Old versions of the application do not update properly

### ISSUE THREAD

**Issue by m133225** [HubTurbo/addressbook#500]
As the updater was still in development. We should probably state this in the documentation/user guide somewhere.

**Comment by damithc** [comment]
Yes, only the recent three releases are compatible with each other.
